npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-assets",
-  "version": "2.5.0-dev",
+  "version": "2.3.0-dev",
```

Comment:
It is 2.3 in the package.json. Did we change it to 2.5 at some point and it got overwritten?